generate_dump.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+
+psql -h "$KNOWLEDGE_GRAPH_HOST" -p "$KNOWLEDGE_GRAPH_PORT" --username=postgres < "$DUMP_NAME"
+psql -d "$KNOWLEDGE_GRAPH_DATABASE" -h "$KNOWLEDGE_GRAPH_HOST" -p "$KNOWLEDGE_GRAPH_PORT" --U "$KNOWLEDGE_GRAPH_USER"
+
+psql -d postgres -h localhost -p 5432 -U postgres -c 'DROP TABLE IF EXISTS kebechet_github_installations'
```

Comment:
is this going to be a cascade operation removing all linked docs? See question below